.github/workflows/joss_paper.yml

```diff
@@ -22,5 +22,5 @@ jobs:
           python-version: 3.9
       - run: pip install -e .
       - run: pip install pytest-codeblocks pytest
-      - run: python -c "import pytest_codeblocks; code=pytest_codeblocks.extract_from_file('paper/paper.md'); f=open('paper.py', 'w'); f.writelines(block.code for block in code); f.close()"
+      - run: python -c "import pytest_codeblocks; code=pytest_codeblocks.extract_from_file('paper/paperv1.md'); f=open('paper.py', 'w'); f.writelines(block.code for block in code); f.close()"
```

Comment:
@slayoo we need to update this then to test code blocks in the v2 paper as well?

Reply:
exactly

Reply:
Some of the "code" I wrote is more like pseudo-code and I wouldn't expect it to actually execute without error (like there are undefined terms, e.g. `r_dry` and `r_wet`). What can be done about this?

Reply:
If making it actually runnable would not bring too much unrelated code into the listing, then I'd vote for making all such snippets actually work. If that is not really feasible, we can include a `.py` file in the `paper` directory with some kind of "preamble" that the `run: python -c "import pytest_codeblocks; code=...` line would prepend to the paper.py file.